tests/integration.rs

```diff
@@ -70,21 +70,22 @@ fn increment() {
     let mut deps = mock_instance(WASM);
 
     let msg = InitMsg { count: 17 };
-    let params = mock_params(
+    let env = mock_env(
         &deps.api,
         "creator",
         &coin("2", "token"),
         &coin("2", "token"),
     );
-    let _res = init(&mut deps, params, msg).unwrap();
+    let _res = init(&mut deps, env, msg).unwrap();
 
     // beneficiary can release it
-    let params = mock_params(&deps.api, "anyone", &coin("2", "token"), &[]);
+    let env = mock_env(&deps.api, "anyone", &coin("2", "token"), &[]);
     let msg = HandleMsg::Increment {};
-    let _res = handle(&mut deps, params, msg).unwrap();
+    let _res = handle(&mut deps, env, msg).unwrap();
 
     // should increase counter by 1
     let res = query(&mut deps, QueryMsg::GetCount {}).unwrap();
+    // get(0).expect("no message")
     let value: CountResponse = from_slice(&res).unwrap();
```

Comment:
running into an issue with res here, 

```
mismatched types

expected slice `[u8]`, found struct `cosmwasm::encoding::Binary`
```

this is the error, i am quite new to rust

Reply:
[The type `Binary`](https://github.com/confio/cosmwasm/blob/v0.7.0/src/encoding.rs#L16-L37) has an `as_slice`, so `let value: CountResponse = from_slice(res.as_slice()).unwrap();` (untested) should work

Reply:
thank you